app/models/loan.rb

```diff
@@ -84,6 +85,10 @@ def member_renewable?
     renewable? && within_borrow_policy_duration? && item.borrow_policy.member_renewable? && ended_at.nil?
   end
 
+  def member_renewal_requestable?
+    renewable? && within_borrow_policy_duration? && ended_at.nil? && !item.active_holds.any? && renewal_request.nil?
```

Comment:
One question about the last bit of this boolean expression- does `renewal_request.nil?` mean that a person can request another renewal if they have one that has already been approved? I think we want there to only be a single renewal request per loan that is either approved or rejected. I haven't thought a lot about this scenario, though, so let me know if there are angles that I'm missing here.

I am also feeling a need to refactor `renewal?` ,`member_renewable?`, and the new `member_renewal_requestable?` a bit, but that can be something we do in a future piece of work.

Reply:
Yeah, that would be the case, but I think a single renewal request might be more intuitive if that makes sense on your end. Agreed on probably clarifying the renewal methods as well, but for now I can clarify the intent on the other method you mentioned and also clean this up so that it only allows one renewal request